.travis.yml

```diff
@@ -15,15 +15,16 @@ jobs:
   include:
     # stage="test" if no stage is specified
     - env: TEST="build"
-      script: sbt headerCheck compile test
+      script: sbt ";headerCheck; + compile; + test"
     # run ci-release only if previous stages passed
     - stage: release
-      script: sbt ci-release
+      script: sbt + ci-release
```

Comment:
`ci-release` doesn't need `+`... it's a command that calls `+publishSigned` and so on.

Reply:
Good point. Done.